Snippets/Raven/Raven_6/MultiTenantSupport.cs

```diff
@@ -0,0 +1,19 @@
+﻿using NServiceBus;
+
+namespace Raven_5
```

Comment:
Is there any reason why Snippets for our new package RavenDB v6 that supports RavenDB 4 uses the namespace Raven_5 as a name?

Reply:
That's a  whoopsie. I'll fix.